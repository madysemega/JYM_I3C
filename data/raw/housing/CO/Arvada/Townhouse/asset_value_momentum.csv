predictions_asset_value_momentum
0.016549134702669437
0.1757090981983361
0.2989622372509984
0.07776665660382531
0.131748995437725
0.25871408865931317
0.005083290949163998
0.18880681949745884
0.12725569989655458
0.029948441395172654
-0.034927002187926666
0.396611076437315
0.08875949509077191
0.24211577477891533
0.09684465177204209
0.3027467080001234
-0.03674476541880485
0.22487512526496997
0.04889246707038039
0.17553355288986014
0.21828460843431904
0.16773029202441364
0.29812118422110195
0.2407524255672158
0.18084376498558183
0.28217171558968873
0.17627388100599653
