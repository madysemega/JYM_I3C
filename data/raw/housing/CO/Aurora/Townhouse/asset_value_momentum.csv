predictions_asset_value_momentum
0.1945544708112436
-0.34805193142240987
-0.24945343955244612
0.09093588688826573
-0.3968663528599535
-0.05358005612026426
-0.5579260286047995
0.17409947808628978
-0.20700637078537892
-0.3007361669252694
-0.22777123200469543
0.2428976280288397
-0.3633309586377392
-0.14836315091888677
-0.4529109603100303
0.6382480779172511
0.31829781294982307
0.3611226557591516
-0.16325486017057525
0.028963628607633667
0.2470270423579994
-0.5745079193723207
-0.18554556244080186
-0.25265604484163
0.34296826006171494
-0.4704230991776175
-0.30748772686951037
