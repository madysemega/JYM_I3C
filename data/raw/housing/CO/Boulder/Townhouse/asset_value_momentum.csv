predictions_asset_value_momentum
-0.023059571456047846
-0.05667082847141054
0.0022966629842279267
0.0607844874873757
0.296753638256197
-0.18150120084154758
-0.1427450492693736
0.061030672743144755
-0.15190399601812143
-0.07347573502819521
-0.07074537227162282
0.018408427261841043
-0.006142762510277251
-0.09308542081859501
0.01983554369686002
0.05730115771290467
0.2123560512300836
-0.29742532663136484
-0.055260677882463624
-0.12975402881204334
-0.07289309288722949
0.054185467548898594
-0.04984504653301938
-0.21920061240783117
-0.0668589578743939
-0.08996116442952636
0.10665444296108094
