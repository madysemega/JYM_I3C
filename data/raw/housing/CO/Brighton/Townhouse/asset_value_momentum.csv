predictions_asset_value_momentum
-0.033775801593256126
-0.19154392819134936
0.18012322682275983
0.1963782212442525
-0.045137922715377136
-0.0006497603432882248
-0.06479201739147501
-0.025419537233266547
-0.058680815879426654
-0.26831762107815293
0.37051143881590715
0.23842667541222157
-0.31708771970513466
0.3157280571784398
0.04501970199244476
0.23088616604877754
0.5888616173532742
0.18611424638053037
-0.028887985527268838
-0.12012044671662606
-0.39482834689566065
-0.5153005689310203
0.3668353533075492
0.04441948977048792
0.12708557889792704
0.02511105990266726
0.07100931845220317
