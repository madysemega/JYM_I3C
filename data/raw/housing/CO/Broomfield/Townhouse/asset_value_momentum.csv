predictions_asset_value_momentum
-0.023502634730683047
0.05510886377024181
0.21646612767903417
0.03720139200589359
-0.009182409530389502
0.12877470571939784
-0.04203809771830879
0.2317637307538402
-0.05691997280292411
0.06173326538931821
0.16590410835813926
0.09588338785397242
0.2884126768673201
0.20259458325162194
0.04544178729753179
-0.03346300757023722
-0.17637743112713392
-0.23498679285832821
-0.01525960404647451
0.031346379759498715
0.29787618482731315
0.09480428797646927
-0.15086610475693668
0.03166387193529577
0.04119278743372757
-0.06906853076964321
0.053155685106534505
