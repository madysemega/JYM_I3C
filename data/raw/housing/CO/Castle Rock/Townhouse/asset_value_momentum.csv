predictions_asset_value_momentum
-0.13592423635784906
-0.18194418222169562
-0.18354923939678375
-0.06490533144517277
-0.1759263190679
-0.1208819531186435
-0.1250862316962811
0.06419698220206732
-0.19824285805939934
-0.19333504447120514
0.23341962530126553
-0.09591851664461293
-0.14451668286591773
-0.08740849416261273
-0.029863300105715126
-0.09237187356767403
-0.11418635384217657
-0.0013109183235236247
-0.18254594735064886
0.19097699869413595
-0.07776861722023927
-0.3000546247862075
0.025343901185177778
-0.20815027101398706
0.06326333170478692
-0.17961921910787867
-0.15207984864489707
