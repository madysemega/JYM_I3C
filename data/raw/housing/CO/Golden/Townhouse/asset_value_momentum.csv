predictions_asset_value_momentum
-0.14179564404374786
-0.6251246583293337
-0.7723137995510081
-0.20937938265212366
-0.34995399820963685
0.09886634912813816
-0.06934084681912052
-0.6175912790594258
-0.05152631936720901
-0.004356739354956919
0.09679862481356609
-0.44045721776162483
-0.3117361012911009
-0.2795617366342498
0.18432125125498275
-0.6567266294676667
-0.5095439792251425
0.11477765217377198
0.08037037696266139
0.09611657612837492
0.08743904124631224
-0.28294200110495576
-0.1479277405083963
-0.3400095230291968
-0.3441323706139639
-0.4386941696790706
0.45961871109955194
